apps/web-giddh/src/app/theme/form-fields/select-multiple-fields/select-multiple-fields.component.html

```diff
@@ -33,7 +35,7 @@
             />
         </mat-chip-grid>
         <mat-autocomplete #auto="matAutocomplete" (optionSelected)="selectOption($event); selectField.value = ''">
-            <mat-option [ngClass]="{'po-options-line': isPurchaseOrder }" *ngFor="let option of fieldFilteredOptions" [value]="option">
+            <mat-option (click)="(name === emailType.To || name === emailType.Cc || name === emailType.Bcc) && $event.stopPropagation()" [ngClass]="{'po-options-line': isPurchaseOrder }" *ngFor="let option of fieldFilteredOptions" [value]="option">
```

Comment:
```suggestion
            <mat-option (click)="$event.stopPropagation() && (name === emailType.To || name === emailType.Cc || name === emailType.Bcc)" [ngClass]="{'po-options-line': isPurchaseOrder }" *ngFor="let option of fieldFilteredOptions" [value]="option">
```

Reply:
Done

Reply:
If the condition is false, $event.stopPropagation() still runs, which might be unnecessary in some scenarios.